tests/gpflow/kernels/test_kernels.py

```diff
@@ -541,3 +541,17 @@ def test_latent_kernels():
     assert len(multioutput_kernel_list[0].latent_kernels) == 1
     assert multioutput_kernel_list[1].latent_kernels == tuple(kernel_list)
     assert multioutput_kernel_list[2].latent_kernels == tuple(kernel_list)
+
+
+def test_combination_LMC_kernels():
+    N, D = 100, 3
+    L1, L2, P = 2, 3, 2
+    kernel_list1 = [Linear(active_dims=[1]), SquaredExponential()]
+    assert len(kernel_list1) == L1
+    kernel_list2 = [SquaredExponential(), Linear(), Linear()]
+    assert len(kernel_list2) == L2
```

Comment:
A bit [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)er:
```suggestion
    N, D, P = 100, 3, 2
    kernel_list1 = [Linear(active_dims=[1]), SquaredExponential()]
    L1 = len(kernel_list1)
    kernel_list2 = [SquaredExponential(), Linear(), Linear()]
    L2 = len(kernel_list2)
```